### PR TITLE
Ahodges/add euc install

### PR DESCRIPTION
**Pull request by lamagnifica**
@nedbat @catong sanity check please

### CODE REVIEW

**Comment by nedbat** [comment]
:+1:

File: en_us/install_operations/source/installation/devstack/install_devstack.rst [hunk 1]
```diff
@@ -10,111 +10,87 @@ This section describes how to install the Open edX developer stack (devstack).
...
 
-.. _Install DevStack:
+Before you install devstack, make sure that you have met the
+:ref:`installation prerequisites<Installation Prerequisites>`.
 
-******************************************************
-Installing Devstack with a Direct Vagrant Box Download
-******************************************************
+You must also ensure that you have the administrator password for your local
+computer. The administrator password is needed to configure NFS (network file
```
Comment (catong) [inline en_us/install_operations/source/installation/devstack/install_devstack.rst:21]: "^the^ NFS"?

File: en_us/install_operations/source/installation/devstack/install_devstack.rst [hunk 2]
```diff
@@ -10,111 +10,87 @@ This section describes how to install the Open edX developer stack (devstack).
...
-configure NFS to allow users to access code directories directly from your
-computer.
+* Use a BitTorrent client to download the Vagrant box file before you
+  install devstack. The first time you start the devstack virtual machine, the
+  Vagrant virtual machine management tool uses the previously downloaded box
+  file.
 
-#. Ensure the ``nfsd`` client is running. Use the ``nfsd status`` command.
+If your internet connection is limited or intermittent, edX recommends that you
+torrent the box file. However, this step is optional.
```
Comment (catong) [inline en_us/install_operations/source/installation/devstack/install_devstack.rst:41]: "torrent" -> "Use torrents to download"??

Comment (catong) [inline en_us/install_operations/source/installation/devstack/install_devstack.rst:41]: Hm, I see that you're using "Torrent" as a verb in the following topic. You can ignore the previous comment, then.

File: en_us/install_operations/source/installation/devstack/install_devstack.rst [hunk 3]
```diff
@@ -10,111 +10,87 @@ This section describes how to install the Open edX developer stack (devstack).
...
+system) to allow users to access code directories directly from your computer.
 
-To install devstack, follow the instructions
-below. During the installation, the Vagrant virtual machine management tool will download an Open edX virtual maching ``box`` file.
+===========================
+Selecting a Download Option
+===========================
 
-.. note::
+The Open edX virtual machine box file has a file size of approximately four
```
Comment (catong) [inline en_us/install_operations/source/installation/devstack/install_devstack.rst:28]: Throughout: "`.box` file"?

File: en_us/install_operations/source/installation/devstack/install_devstack.rst [hunk 4]
```diff
@@ -10,111 +10,87 @@ This section describes how to install the Open edX developer stack (devstack).
...
-   .. code-block:: bash
-
-     mkdir devstack
-     cd devstack
+===============================
+Torrent the Box File (Optional)
+===============================
 
-#. Download the devstack Vagrant file.
+#. Download the torrent file for the latest Open edX release. For the URLs and
```
Comment (catong) [inline en_us/install_operations/source/installation/devstack/install_devstack.rst:50]: "For" -> "To obtain"

File: en_us/install_operations/source/platform_releases/eucalyptus.rst [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+After upgrading Open edX to the Eucalyptus release, start the LMS and Studio and
+verify that course content and data was migrated correctly.
+
+
+********************************************
+Upgrading to a Subsequent Eucalyptus Release
+********************************************
+
+Occasionally, we release updates to Eucalyptus.  For example, the second
```
Comment (catong) [inline en_us/install_operations/source/platform_releases/eucalyptus.rst:119]: The tense of the first sentence is odd. "If updates are released for Eucalyptus, they are given numbers. For example, ..."

**Comment by catong** [comment]
I made some comments that are optional for merging this PR, perhaps consider for a follow-on doc story. 👍

**Pull request merged**